test/integration/red_test.go

```diff
@@ -327,7 +327,7 @@ func testREDMetricsForJavaHTTPLibrary(t *testing.T, url string, comm string, sys
 
 func testREDMetricsJavaHTTP(t *testing.T) {
 	for _, testCaseURL := range []string{
-		"http://localhost:8085",
+		"http://localhost:8084",
```

Comment:
I gave up on filtering the proxies for the tests, so when we look for the PORT now we find the correct service.